Services/include/general.h

```diff
@@ -25,7 +25,11 @@
 
 SAT_returnState start_general_service(void);
 
-typedef enum { REBOOT = 0 } General_Subtype; // shared with EPS!
+typedef enum {
+    REBOOT = 0,
+    GET_UHF_WATCHDOG_TIMEOUT = 1,
+    SET_UHF_WATCHDOG_TIMEOUT = 2,
+} General_Subtype;
```

Comment:
Is this how clang-format formatted this line? I noticed the enum on line 34 is styled differently

Reply:
It appears that clang-format doesn't have a preference in this case, I suppose because the one on 34 is still under the line-character-limit

Reply:
Bit odd that it behaves like that. Let's leave it to avoid conflicts later